index.html

```diff
@@ -162,92 +301,134 @@ <h3 class="project-title">Multi-Post Stories</h3>
+                <h2 class="hex connect" id="connect">LET’S CONNECT</h2>
+                <ul id="platforms">
+                    <div>
+                        <li><a href="#"><img src="images/socials/twitter.svg" alt="twitter"></a></li>
+                        <li><a href="#"><img src="images/socials/Linkedin.svg" alt="Linkedin"></a></li>
+                        <li><a href="#"><img src="images/socials/m.svg" alt="svg"></a></li>
+                        <li><a href="#"><img src="images/socials/Vector1.svg" alt="vectori1"></a></li>
+                        <li><a href="#"><img src="images/socials/Vector2.svg" alt="vectori2"></a></li>
 
-                <li class="frameworks">
-                    <p class="langusge">Skills</p>
-                    <p class="sign">></p>
-                </li>
-            </ul>
+                    </div>
+                    
+                </ul>
+                <ul class="social-media" id="medias">
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/twitter.svg" /></a>
+                    </li>
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+                    </li>
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/m.svg" /></a>
+                    </li>
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/Vector1.svg" /></a>
+                    </li>
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/Vector2.svg" /></a>
+                    </li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">Get my resume</button></a>
+                </div>
+        
+            </div>
+        
+            <div class="languages-list" class="liste-languages">
+                <ul>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="downarrow" src="images/Icons/vector.png" alt="arrow">
+        
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="newlist">
+                        <ul class="new-nested-li">
+                            <li id="java"><img src="images/langusges/js.png" alt="js">
+                                <p>JavaSript</p>
+                            </li>
+                            <li id="html"><img src="images/langusges/html.png" alt="html">
+                                <p>HTML</p>
+                            </li>
+                            <li id="css"><img src="images/langusges/css.png" alt="css">
+                            <p>CSS</p>
+                            </li>
+                        </ul>
```

Comment:
- [ ] kinldy ensure that you add sufficient padding around your list of technologies and also ensure that you use the appropriate font-size, font-weight and color, so that it matches the figma design provided 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222919603-7ee9bc8f-2f7d-4332-a783-919415c32ee9.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/222919650-da0f3e33-1611-4add-8841-bd734793e8ec.png)